errornotifier/middleware.go

```diff
@@ -58,6 +59,9 @@ func NotifyOnPanic(n Notifier, req *http.Request, f func()) (err error) {
 			err = e
 		}
 
+		l := log.ForceContext(req.Context())
+		l.Error().Log("msg", err.Error(), "stack", string(debug.Stack()))
+
```

Comment:
This is a good idea, but the wrong place for the code, because this function is for sending errors to Airbrake:

> NotifyOnPanic will notify Airbrake if function f panics

You want to log the stack when a handler panics? Then `server.Recovery` would probably be a better place to handle this:

https://github.com/theplant/appkit/blob/b0c310bcf0d9e9b5bd20739d97e6fc1306d6d89f/server/recovery.go#L11-L21

Reply:
Actually, isn't this already done here?

https://github.com/theplant/appkit/blob/b0c310bcf0d9e9b5bd20739d97e6fc1306d6d89f/server/logger.go#L56-L66